iOS-Email-Client/Libs/FileUtils.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  FileUtils.swift
+//  iOS-Email-Client
+//
+//  Created by Daniel Tigse on 1/25/19.
+//  Copyright © 2019 Criptext Inc. All rights reserved.
+//
+
+import Foundation
+
+class FileUtils{
+    
+    static func saveEmailBodyToFile(account: Account, metadataKey: String, body: String){
+        let fileUrl = FileUtils.getURLForBody(email: "\(account.username)\(Env.domain)", metadataKey: metadataKey)
+        guard let data = body.data(using: .utf8) else {
+            return
+        }
+        guard FileManager.default.fileExists(atPath: fileUrl.path) else {
+            let directoryUr = FileUtils.getDirectoryURLForEmail(email: "\(account.username)\(Env.domain)", metadataKey: metadataKey)
+            try! FileManager.default.createDirectory(atPath: directoryUr.path, withIntermediateDirectories: true, attributes: nil)
+            try! data.write(to: fileUrl, options: .atomic)
+            return
+        }
+        if let fileHandle = try? FileHandle(forUpdating: fileUrl) {
+            fileHandle.seekToEndOfFile()
+            fileHandle.write(data)
+            fileHandle.closeFile()
+        }
+    }
+    
+    static func saveEmailHeadersToFile(account: Account, metadataKey: String, headers: String){
```

Comment:
`saveEmailBodyToFile ` and `saveEmailHeadersToFile ` have the exact same code except for ONE line, abstract this line for the rest of the code and reuse it

Reply:
TWO lines actually. but OK!